cvmfs/server/cvmfs_server_mkfs.sh

```diff
@@ -67,7 +67,9 @@ cvmfs_server_alterfs() {
   elif ! is_master_replica $name && [ $master_replica -eq 1 ]; then
     echo -n "Allowing Replication of this Repository... "
     local master_replica="${temp_dir}/.cvmfs_master_replica"
-    touch $master_replica
+    # Azurite does not like direct empty file uploads;
+    # touch $master_replica
+    head -c 10 /dev/random > $master_replica
```

Comment:
Let's put something meaningful inside then, e.g. "This file marks the repository as replicatable to stratum 1 servers" or similar

Reply:
done